test/suites/common/tracer.js

```diff
@@ -1,223 +1,231 @@
+            let extractedContext = Tracer.extract(opentracing.FORMAT_HTTP_HEADERS, carrier);
             expect(extractedContext._guid).to.equal(spanContext._guid);
             expect(extractedContext.traceGUID()).to.equal(spanContext.traceGUID());
             expect(extractedContext._sampled).to.equal(spanContext._sampled);
             expect(extractedContext.getBaggageItem('footwear')).to.equal('sandals');
             expect(extractedContext.getBaggageItem('creditcard')).to.equal('visa');
 
-            var span2 = Tracer.startSpan('my_child', { childOf: extractedContext});
+            let span2 = Tracer.startSpan('my_child', { childOf : extractedContext });
             expect(span2.context()._sampled).to.equal(false);
         });
 
-        it("should propagate B3 format (64 bit traceid)", function() {
+        it('should propagate B3 format (64 bit traceid)', () => {
             Tracer._propagators[Tracer._opentracing.FORMAT_HTTP_HEADERS] = new lightstep.B3Propagator(Tracer);
 
             headers = {
-                'x-b3-spanid': '5e7c1e9617cdd152',
-                'x-b3-traceid': '0a0f6c981e2430cb',
-                'x-b3-sampled': '1'
+                'x-b3-spanid'  : '5e7c1e9617cdd152',
+                'x-b3-traceid' : '0a0f6c981e2430cb',
+                'x-b3-sampled' : '1',
             };
 
-            var context = Tracer.extract(opentracing.FORMAT_HTTP_HEADERS, headers);
+            let context = Tracer.extract(opentracing.FORMAT_HTTP_HEADERS, headers);
             expect(context.traceGUID()).to.equal('00000000000000000a0f6c981e2430cb');
 
             carrier = {};
             Tracer.inject(context, opentracing.FORMAT_HTTP_HEADERS, carrier);
             expect(headers['x-b3-traceid']).to.equal('0a0f6c981e2430cb');
         });
 
-        it("should propagate B3 format (128 bit traceid)", function() {
+        it('should propagate B3 format (128 bit traceid)', () => {
             Tracer._propagators[Tracer._opentracing.FORMAT_HTTP_HEADERS] = new lightstep.B3Propagator(Tracer);
 
             headers = {
-                'x-b3-spanid': '5e7c1e9617cdd152',
-                'x-b3-traceid': '10000000000000000a0f6c981e2430cb',
-                'x-b3-sampled': '1'
+                'x-b3-spanid'  : '5e7c1e9617cdd152',
+                'x-b3-traceid' : '10000000000000000a0f6c981e2430cb',
+                'x-b3-sampled' : '1',
             };
 
-            var context = Tracer.extract(opentracing.FORMAT_HTTP_HEADERS, headers);
+            let context = Tracer.extract(opentracing.FORMAT_HTTP_HEADERS, headers);
             expect(context.traceGUID()).to.equal('10000000000000000a0f6c981e2430cb');
 
             carrier = {};
             Tracer.inject(context, opentracing.FORMAT_HTTP_HEADERS, carrier);
             expect(headers['x-b3-traceid']).to.equal('10000000000000000a0f6c981e2430cb');
         });
 
-        it("should propagate binary carriers");
+        it('should propagate binary carriers', () => {
```

Comment:
This is the modified test in here, the rest of the changes were just formatting based on es-lint.